packages/contract-method-name/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@cfxjs/contract-method-name",
+  "type": "module",
+  "main": "index.js",
+  "dependencies": {
+    "eth-method-registry": "2.0.0",
+    "ethjs": "0.4.0",
+    "js-conflux-sdk": "1.6.9",
+    "node-fetch": "2.6.1"
+  },
+  "devDependencies": {
+    "nock": "13.1.1"
```

Comment:
better move nock as the devDependency to the root of helios, since it's used by jest